doc/classes/SpatialMaterial.xml

```diff
@@ -187,12 +224,14 @@
 			The reflectivity of the object's surface. The higher the value, the more light is reflected.
 		</member>
 		<member name="metallic_specular" type="float" setter="set_specular" getter="get_specular" default="0.5">
-			General reflectivity amount.
+			Sets the size of the specular lobe. The specular lobe is the bright spot that is reflected from light sources.
 			[b]Note:[/b] unlike [member metallic], this is not energy-conserving, so it should be left at [code]0.5[/code] in most cases. See also [member roughness].
 		</member>
 		<member name="metallic_texture" type="Texture" setter="set_texture" getter="get_texture">
+			Texture used to specify metallic for an object. This is multiplied by [member metallic].
 		</member>
 		<member name="metallic_texture_channel" type="int" setter="set_metallic_texture_channel" getter="get_metallic_texture_channel" enum="SpatialMaterial.TextureChannel" default="0">
+			Specifies the channel of the [member metallic_texture] in which the metallic information is stored. Lighting modifies the alpha so shadowed areas are opaque and non-shadowed areas are transparent. Useful for overlaying shadows onto a camera feed in AR.
```

Comment:
Is the comment about camera feed in AR meant to be here, or was it wrongly copy pasted from `flags_use_shadow_to_opacity`?

Reply:
Looks like I did this three times. I dont know what I meant to copy-paste, but here we are.